crates/cxx-qt-gen/src/generator/cpp/method.rs

```diff
@@ -21,13 +21,13 @@ use indoc::formatdoc;
 use syn::{spanned::Spanned, Error, FnArg, Pat, PatIdent, PatType, Result};
 
 pub fn generate_cpp_methods(
-    invokables: &Vec<ParsedMethod>,
+    invokables: &Vec<&ParsedMethod>,
     qobject_idents: &QObjectNames,
     type_names: &TypeNames,
 ) -> Result<GeneratedCppQObjectBlocks> {
     let mut generated = GeneratedCppQObjectBlocks::default();
     let qobject_ident = qobject_idents.name.cxx_unqualified();
-    for invokable in invokables {
+    for &invokable in invokables {
```

Comment:
do you need the `&` here?

Reply:
Yeah it's used in the QMethod::try_from, was erroring since the trait isn't implemented for &&ParsedMethod, might be able to put the ref inside that fn call, but I think it makes more sense since invokables is a Vec of references